src/integrate/orchestrator.rs

```diff
@@ -0,0 +1,1204 @@
+    "createwallet",
+    "importaddress",
+    "sendrawtransaction",
+    "getrawtransaction",
+    "scantxoutset",
+    "getrawmempool",
+    "getblockhash",
+]
+# Expedite the key registration / genesis block
+[[blocks]]
+count = 1
+block_time = 30000
+ignore_txs = false
+
+# Expedite the key registration / genesis block
+[[blocks]]
+count = 2
+block_time = 20000
+ignore_txs = false
+
+# Give more time to the first blocks
+[[blocks]]
+count = 3
+block_time = 45000
+ignore_txs = false
+"#,
+            devnet_config.bitcoin_controller_port,
+            devnet_config.bitcoin_controller_block_time,
+            devnet_config.miner_btc_address,
+            devnet_config.bitcoin_node_rpc_port,
+            devnet_config.bitcoin_node_username,
+            devnet_config.bitcoin_node_password,
+        );
+        let mut bitcoin_controller_conf_path = PathBuf::from(&devnet_config.working_dir);
+        bitcoin_controller_conf_path.push("conf/puppet-chain.toml");
+
+        let mut file = File::create(bitcoin_controller_conf_path)
+            .expect("Unable to create bitcoin_controller.toml");
+        file.write_all(bitcoin_controller_conf.as_bytes())
+            .expect("Unable to create bitcoin_controller.toml");
+
+        let mut exposed_ports = HashMap::new();
+        exposed_ports.insert(
+            format!("{}/tcp", devnet_config.bitcoin_controller_port),
+            HashMap::new(),
+        );
+        exposed_ports.insert(
+            format!("{}/tcp", devnet_config.bitcoin_node_rpc_port),
+            HashMap::new(),
+        );
+        exposed_ports.insert(
+            format!("{}/tcp", devnet_config.bitcoin_node_p2p_port),
+            HashMap::new(),
+        );
+
+        let mut labels = HashMap::new();
+        labels.insert("project".to_string(), self.network_name.to_string()); 
+
+        let config = Config {
+            labels: Some(labels),
+            image: Some(devnet_config.bitcoin_node_image_url.clone()),
+            domainname: Some(self.network_name.to_string()),
+            tty: Some(true),
+            exposed_ports: Some(exposed_ports),
+            entrypoint: Some(vec![]),
+            host_config: Some(HostConfig {
+                port_bindings: Some(port_bindings),
+
+                binds: Some(vec![
+                    format!("{}/conf:/etc/bitcoin", devnet_config.working_dir),
+                    format!("{}/data/bitcoin:/root/.bitcoin", devnet_config.working_dir),
+                ]),
+                ..Default::default()
+            }),
+            ..Default::default()
+        };
+
+        let options = CreateContainerOptions {
+            name: format!("bitcoin.{}", self.network_name),
+        };
+
+        let container = docker
+            .create_container::<String, String>(Some(options), config)
+            .await
+            .map_err(|e| format!("Unable to create container: {}", e))?
+            .id;
+
+        self.bitcoin_blockchain_container_id = Some(container.clone());
```

Comment:
Similar to the above comment to print out the image ID, debug or verbose logs for printing this would be helpful for Docker users as well.

Reply:
Addressed with latest